src/commcare_cloud/ansible/roles/nginx/vars/reach_ssl.yml

```diff
@@ -0,0 +1,65 @@
+---
+nginx_ssl_cert: "{{ reach_nginx_ssl_cert }}"
+nginx_ssl_key:  "{{ reach_nginx_ssl_key }}"
+
+nginx_sites:
+- server:
+   balancers:
+    - name: "{{ deploy_env }}_commcare"
+      hosts: webworkers
+      port: "{{ django_port }}"
+    - name: "{{ deploy_env }}_formplayer"
+      hosts: formplayer
+      port: "{{ formplayer_port }}"
+      method: "hash $cookie_sessionid consistent"
+   proxy_cache_path:
+    - name: "hq_media_cache"
+      path: "{{ www_home }}/hq_media/cache"
+      max_size: "2g"
+      inactive_time: "24h"
+   file_name: "{{ deploy_env }}_reach_commcare"
+   listen: "443 ssl http2"
+   server_name: "{{ REACH_SITE_HOST }}"
+   client_max_body_size: 100m
+   proxy_set_headers:
+   - "Host $host"
+   - "X-Forwarded-For $remote_addr"
+   - "X-Forwarded-Protocol  $scheme"
+   add_header: "X-Frame-Options SAMEORIGIN"
+   access_log: "{{ log_home }}/{{ deploy_env }}-timing.log timing"
+   locations:
+    - name: /
+      balancer: "{{ deploy_env }}_commcare"
+      proxy_redirect: "http://{{ REACH_SITE_HOST }} https://{{ REACH_SITE_HOST }}"
+      proxy_next_upstream_tries: 1
+      proxy_read_timeout: 1200s # 900s temporarily increase timeout for UCR data exports for ICDS
```

Comment:
Is comment still applicable?

Reply:
https://github.com/dimagi/commcare-cloud/pull/2580/commits/32c99bdfffbbf9119b41b14c57075c3dbe987d7f